interop/xds/client/client.go

```diff
@@ -368,6 +378,22 @@ func parseRPCMetadata(rpcMetadataStr string, rpcs []string) []*rpcConfig {
 
 func main() {
 	flag.Parse()
+	if *enableCSMObservability {
+		exporter, err := prometheus.New()
+		if err != nil {
+			logger.Fatalf("Failed to start prometheus exporter: %v", err)
+		}
+		provider := metric.NewMeterProvider(
+			metric.WithReader(exporter),
+		)
+		go http.ListenAndServe(":9464", promhttp.Handler())
```

Comment:
Please add port flag and use it here.

Reply:
*or environment variable, whatever is decided.

Reply:
Decided to do env var "OTEL_EXPORTER_PROMETHEUS_PORT" with default of 9464. Interop test won't set and will default.